tests/v2/test_waku_store.nim

```diff
@@ -690,12 +736,16 @@ procSuite "Waku Store":
 
       let successResult = await proto3.resume(some(@[offListenSwitch.peerInfo.toRemotePeerInfo(),
                                                      listenSwitch.peerInfo.toRemotePeerInfo(),
-                                                     listenSwitch.peerInfo.toRemotePeerInfo()]))
+                                                     listenSwitch.peerInfo.toRemotePeerInfo(),
```

Comment:
Wondering why did you add `listenSwitch` twice? if you have a specific test scanrio, please add a small comment